<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsManagerWidget</class>
 <widget class="QWidget" name="PluginsManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbAddPluginsFromRepo">
        <property name="text">
         <string>Add from repo..</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pdAddPluginFromLocal">
        <property name="text">
         <string>Add from local...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
